pkg/controller/plan/kubevirt.go

```diff
@@ -1166,6 +1173,33 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 	return
 }
 
+// Attempt to find a suitable instancetype preference.
+func (r *KubeVirt) instancetype(vm *plan.VMStatus) (virtualMachine *cnv.VirtualMachine, ok bool) {
+	instancetypeName, err := r.Builder.InstanceTypeName(vm.Ref)
+	if err != nil {
+		return
+	}
+
+	instanceTypePreference := &instancetype.VirtualMachineClusterPreference{}
```

Comment:
I see that there's also a namespaced entity (virtualmachinepreferences.instancetype.kubevirt.io) - I guess we should first try to look for that one, but need to check this with kubevirt guys

Reply:
Yes, depending on the destination namespace, there might be "local" instancetypes/preferences there too.

Reply:
added